### DISCUSSION TITLE
How to use this in v2rayN (Windows App)

### DISCUSSION THREAD

**Discussion by robin98** [v2fly/domain-list-community#1651] [Q&A]
Which example is right for use this inside the v2rayN app?

`ext:dlc.dat:category-bank-ir`

OR

`include:category-bank-ir`

OR

`geosite:category-ir-bank`

**Comment by SaeedMalek** [comment]
1 - Back up the original geosite.dat file which is located in the root of your v2rayN folder. It is approx. 1300 KB.
2 - Download dlc.dat and rename it to geosite.dat. Overwrite this file over the original geosite.dat.
3 - In the v2rayN app, select settings, RoutingSetting, double click on your desired rule set (e.g.: Whitelist), select "Rule Add", select "direct" from outboundTag drop-down menu, under domain column type: geosite:category-bank-ir
4 - Push "Confirm" on all windows.
5 - Done.

**Reply by robin98** [reply]
What is the main difference between this repo and the original `geosite.dat` ?
Do you think this is a good idea to overwrite that file, as the original file have more content compare to this one?

**Comment by MkQtS** [comment]
For your original question, use `geosite:category-bank-ir`

v2rayN uses https://github.com/Loyalsoldier/v2ray-rules-dat by default, check the README there for the difference.

BTW, v2rayN supports to specify sources of geo files now, https://github.com/2dust/v2rayN/pull/5829, which may be usefull for non-cn users.
